charts/karavi-metrics-powerflex/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "0.1.0"
+appVersion: "0.1.1"
```

Comment:
Should this be at version 0.2.0 so it matches the upcoming 0.2.0 release of the karavi-topology image release?

Reply:
Yep - have to change that.